README.md

```diff
@@ -16,72 +16,72 @@ Backed by Y Combinator
 
 ---
 
-**Fern is an open source format for defining REST APIs.**
-You can think of it like a programming language to describe
-your API: your endpoints, types, errors, and examples.
+# Fern
 
-This repository contains the **Fern compiler.** The compiler transforms the API description into useful outputs, like:
+**Fern** is an **open source toolkit** for defining, building and generating **APIs, documentation and SDKs**. Bring an existing OpenAPI definition or describe your endpoints in a Fern specification. Fern generates backend code, specifications in various formats, and client libraries with a single command.
 
-### 🌿 SDKs
+This repository contains the Fern **compiler**. The compiler transforms an API definition into various code artifacts, such as client libraries, SDKs, or even full-fledged **Express**, **Fast API** or **Spring Boot** backends.
 
-Client libraries speed up internal development, and help acquire customers who use your API. Our auto-generated SDKs are idiomatic and feel handwritten.
+In short, Fern takes care of the boilerplate associated with developing APIs, so you can **focus on delivering business value**.
 
-### 🌿 Server-side code generation
+# Capabilities
```

Comment:
Versioning an API, SDKs, and docs comes up often in customer calls. Where do you think we ought to mention 1-2 sentences about versioning?

Reply:
I think versioning isn't an important enough topic to include the README, i can see us leaving that in the docs somewhere. (and fern doesn't _really_ solve this problem since its pretty unopinionated about versioning)